recipes/jupyterlab-nbconvert-nocode/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "jupyterlab_nbconvert_nocode" %}
+{% set version = "0.3.1" %}
+{% set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7c1617d2b49d425984d30e5e0cc9e7e1b354bb570dab474cafffb70f61b5162
+
+build:
+  number:  {{ build_num }}
+
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . --no-deps -vv
+      number: {{ build_num }}
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+        - jupyter-packaging
+      run:
+        - python >=3.6
+        - jupyterlab >=1.0.0
+        - nbconvert >=6.0.0
+
+    test:
+      imports:
+        - jupyterlab_nbconvert_nocode
+      commands:
+        - pip check
+      requires
```

```suggestion
      requires:
```
Missed that, my mistake.

---

ah good catch! I was looking for what it was